bin/run-ua-from-snap

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+set -u
+
+export PYTHONPATH="$SNAP"/usr/lib/python3/dist-packages
+"$SNAP"/usr/bin/ubuntu-advantage "$@"
```

Comment:
I think in this scenario it's going to use the system python instead of the one in the Subiquity snap.
We have been setting the PYTHON env variable, which can be done in the snapcraft file.
See this example:
https://github.com/canonical/subiquity/blob/main/snapcraft.yaml#L14

If you do that, do you need the wrapper script?  I though it picked up dist-packages automatically and that it was the site packages causing the problem.

Reply:
Thanks for the feedback.
You are right, the following construct picks up the right version of Python and `ubuntu-advantage`:

```bash
$PYTHON "${SNAP}"/usr/bin/ubuntu-advantage --version
```
I guess I can get rid of the wrapper script.

Reply:
Done! I'm not forcing the use of the Python installation from the snap.